src/device-registry/utils/get-device-details.js

```diff
@@ -18,6 +18,7 @@ const getDetail = async (
   loc,
```

Comment:
We shall need to review this util function to ensure that this change in its parameters does not break anything in any other piece of code which utilises it.

I guess this screams the need to fast track the unit test implementations for this microservice so that any issues are captured automatically and not manually. :)

Reply:
thanks @Baalmart, enforcing TDD/unit tests for this microservice shall commence next sprint.